assets/js/state/checksSelection.js

```diff
@@ -5,6 +5,10 @@ const SAVING = 'SAVING';
 const SUCCESSFULLY_SAVED = 'SUCCESSFULLY_SAVED';
 const SAVING_FAILED = 'SAVING_FAILED';
 
+export const isSaving = (status) => SAVING === status;
+export const isSuccessfullySaved = (status) => SUCCESSFULLY_SAVED === status;
+export const isSavingFailed = (status) => SAVING_FAILED === status;
+
```

Comment:
The idiomatic way to do this is creating selectors that return boolean values. `reselect` would help, but we still didn't introduce that.

Reply:
Cool! Would you mind tracking that so we can plan for a refactor?

Reply:
Yes absolutely! Tracking that on Jira right now, could you still move this to selectors and base the functions on `resourceID`?

Reply:
sure. You meant `targetID` I guess :smile:

Reply:
`whateverID`, I just get lost into these names :joy: